components/suggestion/switch_channel_provider.jsx

```diff
@@ -109,8 +116,11 @@ class SwitchChannelSuggestion extends Suggestion {
 
 function mapStateToPropsForSwitchChannelSuggestion(state, ownProps) {
     const channelId = ownProps.item && ownProps.item.channel ? ownProps.item.channel.id : '';
+    const draft = getPostDraft(state, StoragePrefixes.DRAFT, channelId);
```

Comment:
Since you're using this in both places, it could make a good helper function if you wanted to change that :D